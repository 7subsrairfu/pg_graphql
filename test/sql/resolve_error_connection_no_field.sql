select gql.resolve($$
{
  allAccounts {
    dneField
    totalCount
  }
}
$$);
