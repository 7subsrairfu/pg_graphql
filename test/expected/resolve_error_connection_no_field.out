select gql.resolve($$
{
  allAccounts {
    dneField
    totalCount
  }
}
$$);
                                      resolve                                       
------------------------------------------------------------------------------------
 {"data": null, "errors": ["Unknown field 'dneField' on type 'AccountConnection'"]}
(1 row)

